packages/accounts/plugindefs/multi-owner/config.ts

```diff
@@ -0,0 +1,10 @@
+import { sepolia } from "viem/chains";
+import type { PluginGenConfig } from "../types";
+import { MultiOwnerPluginAbi } from "./abi.js";
+
+export const MultiOwnerPluginGenConfig: PluginGenConfig = {
+  name: "MultiOwnerPlugin",
+  abi: MultiOwnerPluginAbi,
+  address: "0x56bC629F342821FBe91C5273880792dFECBE7920",
```

Comment:
nit: should we make this a util method i.e. `getDefaultMSCAFactoryAddress(chain)`

Reply:
this is a different address, it's the plugin address and we will only ever use one chain here. Also until this is in production we shouldn't introduce utils for getting default addresses